tool/tctl/common/token_command.go

```diff
@@ -78,6 +87,9 @@ func (c *TokenCommand) Initialize(app *kingpin.Application, config *service.Conf
 	c.tokenAdd.Flag("ttl", fmt.Sprintf("Set expiration time for token, default is %v hour, maximum is %v hours",
 		int(defaults.SignupTokenTTL/time.Hour), int(defaults.MaxSignupTokenTTL/time.Hour))).
 		Default(fmt.Sprintf("%v", defaults.SignupTokenTTL)).DurationVar(&c.ttl)
+	c.tokenAdd.Flag("app-name", "Name of the application to add").Default("<Application Name>").StringVar(&c.appName)
```

Comment:
Hmm wouldn't default values like this screw something up? I.e. if you run `token add` command, the default values will be `<Application Name>` and `<URI ...>` which are not valid name/URI?

Reply:
@benarent Do you think we should fail here instead of returning placeholders?

Reply:
Yeah, I agree with Roman, How about we do something like this?

### Before
```log
> teleport start \
   --roles=app \
   --token=fb0aeb98a0c0ca98833901e6f33d47ce \
   --ca-pin=sha256:9695484561b341fb68c1e29c47628a3088b33f7bdfde4438beda8f42f56b870f \
   --auth-server=172.31.13.139:3025 \
   --app-name=<Application Name> \
   --app-uri=<URI of Application> \
   --app-public-addr=<Public Address of Application>

Please note:

  - This invitation token will expire in 60 minutes.
  - 172.31.13.139:3025 must be reachable from the new application service..
  - Update DNS to point <Public Address of Application> to the Teleport proxy.
  - Add a TLS certificate for <Public Address of Application> to the Teleport proxy.
```

### After
It's a little tough to guess this, but I've created a more general example, that's at least partly setup. 

```log
> teleport start \
   --roles=app \
   --token=fb0aeb98a0c0ca98833901e6f33d47ce \
   --ca-pin=sha256:9695484561b341fb68c1e29c47628a3088b33f7bdfde4438beda8f42f56b870f \
   --auth-server=172.31.13.139:3025 \
   --app-name=example-app \
   --app-uri=localhost:8000 \
   --app-public-addr=example-app.[public_addr](if set)

Please note:

  - This invitation token will expire in 60 minutes.
  - 172.31.13.139:3025 must be reachable from the new application service..
  - Update DNS to point example-app.[public_addr](if set) to the Teleport proxy.
  - Add a TLS certificate for example-app.[public_addr](if set) to the Teleport proxy.
```

Reply:
```
Fill out and run this command on a server to make an to make the application available:

> teleport start \
   --roles=app \
   --token=f6f43bff6a6981515cce8010520cc5d7 \
   --ca-pin=sha256:bc7356ed484e8390ef4468bf6cb46738a799ea0b993e734335288e73e1abd498 \
   --auth-server=proxyPublicAddr:3080 \
   --app-name=example-app \               # Change "app-name" to the name of your application.
   --app-uri=http://localhost:8080 \      # Change URI to the address of your internal application.

Please note:

  - This invitation token will expire in 60 minutes.
  - proxyPublicAddr:3080 must be reachable from the new application service..
  - Update DNS to point example-app.proxyPublicAddr to the Teleport proxy.
  - Add a TLS certificate for example-app.proxyPublicAddr to the Teleport proxy under "https_keypairs".
```